sdk/python/setup.py

```diff
@@ -117,16 +117,23 @@
     "pre-commit",
     "assertpy==1.1",
     "pip-tools",
-] + GCP_REQUIRED + REDIS_REQUIRED + AWS_REQUIRED
+    *GCP_REQUIRED,
```

Comment:
Hi @danilopeixoto, what purpose of this change? isn't `a + b + c` syntax easier to read?

Reply:
These changes were really aesthetic. I can reverse those changes if it's an issue (not clear to read).

I changed due to symmetry. Eg:

```python
CI_REQUIRED = [
    "cryptography==3.3.2",
    "flake8",
    ...,
    *GCP_REQUIRED,
    *REDIS_REQUIRED,
    *AWS_REQUIRED,
]
```
vs

```python
CI_REQUIRED = [
    "cryptography==3.3.2",
    "flake8",
    ...,
] + GCP_REQUIRED + REDIS_REQUIRED + AWS_REQUIRED
```

Reply:
Yes, can you revert this please? @danilopeixoto

Reply:
Yes! Reverted to plus sign.